src/jdk.zipfs/share/classes/module-info.java

```diff
@@ -260,12 +260,53 @@
  *           <li>
  *               If the value does not represent a valid
  *               {@linkplain Runtime.Version Java SE Platform version number},
- *               an {@code IllegalArgumentException} will be thrown.
+ *               an {@code IllegalArgumentException} will be thrown when creating
+ *               the ZIP file system.
  *           </li>
  *       </ul>
  *   </td>
  * </tr>
- *  </tbody>
+ * <tr>
+ *   <th scope="row">accessMode</th>
+ *   <td>{@link java.lang.String}</td>
+ *   <td>null/unset</td>
+ *   <td>
+ *       A value defining the desired access mode of the file system.
+ *       ZIP file systems can be created to allow for <em>read-write</em> or
+ *       <em>read-only</em> access.
+ *       <ul>
+ *           <li>
+ *               If no value is set, the file system is created <em>read-write</em>
+ *               if possible. Use {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} to determine the actual access mode.
+ *           </li>
+ *           <li>
+ *               If the value is {@code "readOnly"}, the file system is created
+ *               <em>read-only</em>, and {@link java.nio.file.FileSystem#isReadOnly()
+ *               isReadOnly()} will always return {@code true}. Creating a
+ *               <em>read-only</em> file system requires the underlying ZIP file to
+ *               already exist.
+ *               Specifying {@code create} as {@code true} and {@code accessMode} as
+ *               {@code readOnly} will cause an {@code IllegalArgumentException}
+ *               to be thrown when creating the ZIP file system.
```

Comment:
I 'think' this should be "cause a" or "result in a" for the usages  of "cause an" based on how the phrase is used